### ISSUE TITLE
Your train list

### ISSUE THREAD

**Issue by HansonXia** [open-vision-language/oven#1]
I have read your paper, and your experiments did not train for a complete epoch. Could you please provide the training list you used?

**Comment by edchengg** [comment]
Hi, Thanks for your interest. 
We trained on a randomly shuffled training set. Unfortunately, we don't have the exact training list.

**Comment by HansonXia** [comment]
I attempted fine-tuning on LLaVA and InstructBLIP (blip2_vicuna_instruct & blip2_t5_instruct), but the degradation issue is quite apparent. The test results for early stopping strategies are filled with randomness and do not follow the result curve presented in the paper. Currently, I haven't found a good solution. Do you have any suggestions?

**Comment by edchengg** [comment]
I observe a similar trend.
I tried fine-tuning only the *Q-former layer* of blip2_t5_pretrain with 100 steps and fine-tuning a model on entity split only or query split only (lr=5e-5). 

My results below:

```
===== BLIP2 Zeroshot ====
===== Validation ========
===== Final score 7.87
===== Query Split score 20.58
===== Entity Split score 4.87
===== Query Seen Accuracy 24.63
===== Query Unseen Accuracy 17.68
===== Entity Seen Accuracy 8.55
===== Entity Unseen Accuracy 3.4

===== BLIP2 99 ====
===== Validation ========
===== Final score 9.65
===== Query Split score 20.66
===== Entity Split score 6.29
===== Query Seen Accuracy 26.2
===== Query Unseen Accuracy 17.06
===== Entity Seen Accuracy 11.74
===== Entity Unseen Accuracy 4.3
```

**Comment by HansonXia** [comment]
> I observe a similar trend. I tried fine-tuning only the _Q-former layer_ of blip2_t5_pretrain with 100 steps and fine-tuning a model on entity split only or query split only (lr=5e-5).
> 
> My results below:
> 
> ```
> ===== BLIP2 Zeroshot ====
> ===== Validation ========
> ===== Final score 7.87
> ===== Query Split score 20.58
> ===== Entity Split score 4.87
> ===== Query Seen Accuracy 24.63
> ===== Query Unseen Accuracy 17.68
> ===== Entity Seen Accuracy 8.55
> ===== Entity Unseen Accuracy 3.4
> 
> ===== BLIP2 99 ====
> ===== Validation ========
> ===== Final score 9.65
> ===== Query Split score 20.66
> ===== Entity Split score 6.29
> ===== Query Seen Accuracy 26.2
> ===== Query Unseen Accuracy 17.06
> ===== Entity Seen Accuracy 11.74
> ===== Entity Unseen Accuracy 4.3
> ```

Thank you for your reply. Based on my experiments, blip2_vicuna_instruct yields the best results. I used vicuna7b and fine-tuned vit and qformer, I haven't experimented with larger vicuna models.
